script/tool/lib/src/version_check_command.dart

```diff
@@ -453,6 +458,11 @@ ${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
     if (path.isEmpty) {
       return '';
     }
-    return packagesDir.fileSystem.file(path).readAsStringSync();
+    final File file = packagesDir.fileSystem.file(path);
+    if (!file.existsSync()) {
+      printError('${indentation}No such file: $path');
+      throw ToolExit(_exitMissingChangeDescriptionFile);
```

Comment:
Yeah, since this file (and its path) are managed by the cirrus.yaml, it makes more sense that this is a tool exit error, good call!